modules/govuk/manifests/app/package.pp

```diff
@@ -55,9 +56,13 @@
 
   # Otherwise, assume the apps are checked out straight into /var/govuk.
   } else {
+    if $repo_name == undef {
```

Comment:
Tests are failing, I think because the new parameter is mandatory. I think it's better to do   $repo_name = $title in the define (), so it always takes a default value